v2/src/application.tsx

```diff
@@ -28,6 +28,7 @@ export interface Props {
 
     onEnhetChange(enhet: string): void;
     contextholder?: true | Contextholder;
+    autoSubmit?: boolean;
```

Comment:
kunne eventuellt kalle den `autoSubmitOnMount` for å klargjøre bruken..?

Reply:
Jupp, det kan vi. :)